arxiv/base/templates/base/testmacros.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base/base.html" %}
+
+{% import "base/macros.html" as macros %}
+
+{% block addl_head %}
+
+{% endblock %}
+
+{% block content %}
+
+{{ macros.abs(
```

Comment:
Just curious, are certain fields like `msc_class`, `acm_class`, `journal_ref` missing in the call here because they're not in the example?

Reply:
Yes